app/src/main/res/layout/top_sites_header.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android" android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <TextView
```

Comment:
Re your constraint question -> you could just add basic constraints like 
```
app:layout_constraintBottom_toBottomOf="parent"
app:layout_constraintStart_toStartOf="parent"
app:layout_constraintTop_toTopOf="parent" 
```

Will be helpful to keep it in a `ConstraintLayout` in case we want to add a menu item like the other headers